Standards/scs-XXXX-vN-security-groups-decision-record.md

```diff
@@ -0,0 +1,130 @@
+| 97507e4c-7413-4d61-ab21-    | security_group | 110b1bea-f0e0-4fb7-9fc7-    | access_as_shared   |
+| 047fc23516dd                |                | cda1b6f927b0                |                    |
+| bc22a865-46f9-4cd2-80af-    | security_group | 5f22e42e-95dc-4c0a-8651-    | access_as_shared   |
+| 3c249ba0e010                |                | 57b1dfc8639f                |                    |
+| 2467806f-5428-4506-8e23-    | network        | 02ef8579-9917-4a01-893d-    | access_as_shared   |
+| f4690db04e01                |                | cb2f9f3d5f98                |                    |
+| ed40996e-218d-4daa-b222-    | network        | 73edb86b-d7ab-4db3-82b7-    | access_as_external |
+| f3c603a5b8a6                |                | 25fa8b012e40                |                    |
+| 45e0a707-af82-42a6-b93f-    | subnetpool     | cd7addd1-05d9-48a8-bc38-    | access_as_shared   |
+| efde18216f13                |                | 4a581354983f                |                    |
+| e514c2c8-65ac-4062-8b03-    | subnetpool     | ad1cc1ed-3261-4df2-8c73-    | access_as_shared   |
+| fe24f6fc4656                |                | c3dd72d59061                |                    |
++-----------------------------+----------------+-----------------------------+--------------------+
+stack@devstack:~/devstack$ openstack network rbac show bc22a865-46f9-4cd2-80af-3c249ba0e010
++-------------------+--------------------------------------+
+| Field             | Value                                |
++-------------------+--------------------------------------+
+| action            | access_as_shared                     |
+| id                | bc22a865-46f9-4cd2-80af-3c249ba0e010 |
+| object_id         | 5f22e42e-95dc-4c0a-8651-57b1dfc8639f |
+| object_type       | security_group                       |
+| project_id        | 15f2ab0eaa5b4372b759bde609e86224     |
+| target_project_id | *                                    |
++-------------------+--------------------------------------+
+```
+
+The biggest downside: As soon as a security group is shared, everyone from every project, can edit the rules of this group.
+
+[^1]: [Neutron RBAC](https://docs.openstack.org/neutron/latest/admin/config-rbac.html)
+
+### Option 2: stay project-scoped
+
+Using and adhering the project scope of the security groups has the consequence, that:
+
+1. either an admin has to set up security groups for each project
+2. or the scs only provides a guide on how to setup and use some recommended security groups.
+
+As users are allowed to, will and should edit their security groups, there is no way to ensure, that a certain set of security groups with certain rules is always present in a project.
+So packing an extra burden on admins is unreasonable.
+The option to give a guide and recommend a few security groups however is a quite good option.
+
+## Decision
+
+To be decided.
```

Comment:
We need a decision here. I prefer option 2, writing a guide.

Reply:
I also prefer option 2 here since shared security groups are still mutable at runtime affecting all consuming projects and resources immediately if any changes to them are applied. This sounds problematic from various angles.

Reply:
I wrote a first version of the decision, as everyone is for Option 2 maybe mixed with Option 3